pkg/development/development_test.go

```diff
@@ -109,3 +110,33 @@ definition document {
 
 	require.Equal(t, "document:somedoc#viewer:\n- '[user:someuser[...]] is <document:somedoc#viewer>'\n", generated)
 }
+
+func TestDevContextV1Service(t *testing.T) {
+	defer goleak.VerifyNone(t, goleak.IgnoreTopFunction("github.com/golang/glog.(*loggingT).flushDaemon"), goleak.IgnoreCurrent())
```

Comment:
where is that glog goroutine coming from?

Reply:
Its the goroutine used under the covers by zerolog to log things.

Reply:
that's weird. I've written tests that bring up a full blown server and I didn't have to add those ignores 😕

Reply:
They are only needed if you're tracking remaining goroutines via `goleak`

Reply:
as I said, I have tests running with `goleak` that start up spicedb, and do not have that ignore, so I wondered what was the difference